terms/crossvalidation_lmd.tex

```diff
@@ -0,0 +1,82 @@
+
+\item \(k\)-fold\newline
+This method randomly splits the original data set into \(k\) equally sized subsets, as shown in Fig. \ref{fig:kfold}. The function is then trained and validated \(k\) times, each time taking a different subset as the test data and the remaining \((k-1)\) subsets as the training data, using each of the \(k\) subsets as the test set once. The \(k\) results are averaged to produce a single estimation. Stratified k-fold cross validation is a refinement of the k-fold method, which splits the original samples into equally sized and distributed subsets, having the same proportions of the different target labels \cite{kohavi1995study}.
+
+\begin{figure}
+\centering
+  \includegraphics[width=0.5\linewidth]{kfold.png}
+  \caption{k-Fold Cross Validation where k=4}
+  \label{fig:kfold}
+\end{figure}
+
+\item Repeated random sub-sampling\newline
+This method is also known as the Monte Carlo CV. It splits the data set randomly with replacement into training and test subsets using some predefined split percentage, for every run. Therefore, this generates new training and test data for each run but the test data of the different runs might contain repeated samples, unlike that of \(k\)-fold \cite{xu2001monte}.
+\end{itemize}
+
+All of the above cross-validation methods are used to check whether the model has been overfitted or underfitted and hence estimating the model's ability of fitting to independent data . Such ability is measured using quantitative metrics appropriate for the model and data \cite[-3\baselineskip]{kohavi1995study, arlot2010survey}. In the case of classification problems, the misclassification error rate is usually used whilst for regression problems, the mean squared error (MSE) is usually used. MSE is represented by Eq. \ref{mse}, where n is the total number of test samples, \(Y_i\) is the true value of the \(i^{th}\) instance and \(\hat{Y}_i\) is the predicted value of the \(i^{th}\) instance.
+
+\begin{equation}\label{mse}
+MSE = \frac{1}{n}\sum^{n}_{i=1}(Y_i - \hat{Y}_i)^2
+\end{equation}
+
+Underfitting is when the model has a low degree and so is not flexible enough to fit the data making the model have a low variance and high bias \cite{baumann2003cross}, as seen in Fig. \ref{fig:models}a. Variance is the model's dependence on the training data and bias is model's assumption about the shape of the data \cite{arlot2010survey}. On the other hand, as seen in Fig. \ref{fig:models}b, overfitting is when the model has a too high degree causing it to exactly fit the data as well as the noise and so lacks the ability to generalize \cite{baumann2003cross}, making the model have a high variance. Cross-validation helps reduce this bias and variance since it uses most of the data for both fitting and testing and so helps the model learn the actual relationship within the data. This makes cross-validation a good technique for models to acquire a good bias-variance tradeoff \cite{arlot2010survey}.
```

Comment:
"has a low degree" ?? of what.  cannot understand. 
Always use ~ for non breaking space between Fig.~\ref...